services/core-web/src/components/Home.js

```diff
@@ -80,10 +83,14 @@ export class Home extends Component {
 
   loadStaticContent = () => {
     this.props.dispatch(showLoading());
-    const staticContentActionCreators = Object.getOwnPropertyNames(staticContent).filter(
-      (property) => typeof staticContent[property] === "function"
-    );
-    staticContentActionCreators.forEach((action) => this.props.dispatch(staticContent[action]()));
+    this.props.dispatch(loadBulkStaticContent());
+    this.props.dispatch(fetchInspectors());
+    //  this.props.dispatch(fetchMineDisturbanceOptions());
```

Comment:
Why are we fetching Inspectors here? 
also comments can be deleted

Reply:
inspectors is a query the front end treats as static content, but the backend doesn't think is static content, so it's special.